spin/options.go

```diff
@@ -11,4 +11,5 @@ type Options struct {
 	SpinnerStyle style.Styles `embed:"" prefix:"spinner." set:"defaultForeground=212" envprefix:"GUM_SPIN_SPINNER_"`
 	Title        string       `help:"Text to display to user while spinning" default:"Loading..." env:"GUM_SPIN_TITLE"`
 	TitleStyle   style.Styles `embed:"" prefix:"title." envprefix:"GUM_SPIN_TITLE_"`
+	Location     string       `helo:"Location of spinner" short:"l" type "location" enum:"left,right" default:"left" env:"GUM_SPIN_LOCATION"`
```

Comment:
Small typo: `helo` -> `help`.

Reply:
Also noticed a missing colon after the `type` struct tag. Fixing.

Reply:
argh, sorry about that, and thanks for fixing.